homeassistant/components/sensor/isy994.py

```diff
@@ -1,95 +1,310 @@
 """
-Support for ISY994 sensors.
+Support for ISY994 binary sensors.
 
 For more details about this platform, please refer to the documentation at
-https://home-assistant.io/components/isy994/
+https://home-assistant.io/components/binary_sensor.isy994/
 """
 import logging
+from typing import Callable  # noqa
 
-from homeassistant.components.isy994 import (
-    HIDDEN_STRING, ISY, SENSOR_STRING, ISYDeviceABC)
-from homeassistant.const import (
-    STATE_CLOSED, STATE_HOME, STATE_NOT_HOME, STATE_OFF, STATE_ON, STATE_OPEN)
-
-DEFAULT_HIDDEN_WEATHER = ['Temperature_High', 'Temperature_Low', 'Feels_Like',
-                          'Temperature_Average', 'Pressure', 'Dew_Point',
-                          'Gust_Speed', 'Evapotranspiration',
-                          'Irrigation_Requirement', 'Water_Deficit_Yesterday',
-                          'Elevation', 'Average_Temperature_Tomorrow',
-                          'High_Temperature_Tomorrow',
-                          'Low_Temperature_Tomorrow', 'Humidity_Tomorrow',
-                          'Wind_Speed_Tomorrow', 'Gust_Speed_Tomorrow',
-                          'Rain_Tomorrow', 'Snow_Tomorrow',
-                          'Forecast_Average_Temperature',
-                          'Forecast_High_Temperature',
-                          'Forecast_Low_Temperature', 'Forecast_Humidity',
-                          'Forecast_Rain', 'Forecast_Snow']
-
-
-def setup_platform(hass, config, add_devices, discovery_info=None):
-    """Setup the ISY994 platform."""
-    # pylint: disable=protected-access
-    logger = logging.getLogger(__name__)
-    devs = []
-    # Verify connection
+from homeassistant.components.isy994 import (ISYDevice, SENSOR_NODES, ISY)
+from homeassistant.const import (TEMP_CELSIUS, TEMP_FAHRENHEIT, STATE_OFF,
+                                 STATE_ON)
+from homeassistant.helpers.typing import ConfigType
+
+_LOGGER = logging.getLogger(__name__)
+
+UOM_FRIENDLY_NAME = {
```

Comment:
Ok, as long as it is PR'ed there we can keep it here for the moment (allows us to get it in 0.28 :smile:), but in the long run it doesn't really belong here